packages/rest-typings/src/v1/channels.ts

```diff
@@ -125,4 +125,14 @@ export type ChannelsEndpoints = {
       messages: IMessage[];
     };
   };
+  "channels.addAll": {
+    POST: (params: { roomId: string; activeUserOnly?: boolean }) => {
+      channel: IRoom;
+    };
+  };
+  "channels.anonymousread": {
+    GET: (params: { roomId?: string; roomName?: string }) => {
+      messages: IMessage[];
+    };
```

Comment:
wrong params and missing  results and request props, see` PaginatedRequest` and `PaginatedResult`